frequently-asked-questions.md

```diff
@@ -31,9 +31,9 @@ client.on("ready", () => {
 client.on("ready", () => {
     client.user.setPresence({
         activities: [{ 
-            name: 'my code',
-            type: 'WATCHING'
-        }],
+          name: 'my code',
+          type: 'WATCHING'
+          }],
```

Comment:
Why was the last line given more room here? Usually its in line with the top object.
```suggestion
          name: 'my code',
          type: 'WATCHING'
        }],
```